realm/src/main/java/io/realm/RealmQuery.java

```diff
@@ -1042,10 +1169,36 @@ public double averageFloat(String fieldName) {
     // Min
 
     /**
+     * Find the minimum value of a field.
+     *
+     * @param fieldName   The field to look for a minimum on. Only int, float, and double
```

Comment:
Return value for all integer(byte, short, int, long) field is an instance of Long, not its wrapper type.
It is a bit confusing for users. 

I think it's better to describe about it in @return comment.